tests/install_upgrade_operators/node_component/conftest.py

```
@@ -227,12 +230,32 @@ def hyperconverged_resource_before_np(admin_client, hco_namespace, hyperconverge
+        for sample in TimeoutSampler(
+            wait_timeout=TIMEOUT_2MIN,
+            sleep=TIMEOUT_10SEC,
+            func=assert_nodes_in_healthy_condition,
+            exceptions_dict={NodesNotHealthyConditionError: []},
+            nodes=nodes,
+            healthy_node_condition_type=None,
+        ):
+            if sample is None:
+                break
+    except TimeoutExpiredError:
+        LOGGER.error("Nodes are not healthy after 2 minutes, proceeding may cause cascading failures")
```

Ruff (0.16.0)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)